docs/source/app-configuration.rst

```diff
@@ -0,0 +1,157 @@
+  --timeout 80 \
+  --memory 128
+
+
+=========================  ===========
+Flag                       Description
+-------------------------  -----------
+``--type``                 Type of app integration function being configured. Current choices are: `duo_auth`, `duo_admin`
+``--cluster``              Applicable cluster this function should be configured against.
+``--name``                 Unique name to be assigned to this app integration function. This is useful when configuring multiple accounts per service.
+``--interval``             The interval, defined using a 'rate' expression, at which this app integration function should execute. See AWS Schedule `Rate Expressions <http://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html#RateExpressions>`_.
+``--timeout``              The AWS Lambda function timeout value, in seconds. This should be an integer between 10 and 300.
+``--memory``               The AWS Lambda function max memory value, in megabytes. This should be an integer between 128 and 1536.
+=========================  ===========
+
+.. note:: Duo Security's Admin API is limited to two (2) requests per-minute. Therefore, setting the ``--timeout`` flag to any value between 10 and 60 will be of no additional value. A recommended timeout of 80 seconds will guarantee four (4) requests happen per-execution.
+
+2. Enter the required authentication information
+````````````````````````````````````````````````
+
+The above command will result in a few prompts asking for the required authentication information needed to configure this StreamAlert App.
+
+.. note:: After the last required authentication value is entered, the values are sent to AWS SSM's `Parameter Store <https://aws.amazon.com/ec2/systems-manager/parameter-store/>`_ as a ``SecureString`` to be used as part of this App's config. Due to this requirement, please ensure you have the correct and valid AWS credentials loaded before continuing.
+
+Example Prompts for Duo Auth
+''''''''''''''''''''''''''''
+
+.. code-block:: bash
+
+  Please supply the API URL for your duosecurity instance. This should be in a format similar to 'api-abcdef12.duosecurity.com': api-abcdef12.duosecurity.com
+
+  Please supply the secret key for your duosecurity Admin API. This should a string of 40 alphanumeric characters: 123424af2ae101d47d9704b783c940dffa825678
+
+  Please supply the integration key for your duosecurity Admin API. This should be in a format similar to 'DIABCDEFGHIJKLMN1234': DIABCDEFGHIJKLMN1234
+
+
+Once the above is completed, a logger statement similar to the following will confirm the configuration::
+
+  StreamAlertCLI [INFO]: App authentication info successfully saved to parameter store.
+  StreamAlertCLI [INFO]: Successfully added 'duo_prod_collector' app integration to 'conf/clusters/prod.json' for service 'duo_auth'.
+
+
+And the ``conf/clusters/prod.json`` file will be updated to include the configuration for this App:
```

Comment:
108-145: I think its too much in the weeds for a user. Maybe just say `Your configuration files (conf/clusters/{cluster} and conf/sources.json) have now been updated and are ready to be deployed.`